environments/test/terragrunt.hcl

```diff
@@ -0,0 +1,19 @@
+include "root" {
+  path = find_in_parent_folders()
+  expose = true
+}
+
+terraform {
+  source = "somegit"
```

Comment:
this will get changed to a tag in this repo once we are getting close to release, for now please use https://terragrunt.gruntwork.io/docs/features/keep-your-terraform-code-dry/#working-locally to override this dummy setting and deploy from local code directly